futures-core/src/stream_obj.rs

```diff
@@ -0,0 +1,161 @@
+//! Asynchronous stream object value
+
+use std::fmt;
+use std::marker::{PhantomData,Unpin};
+use std::task::{ Context, Poll };
+use core::mem::PinMut;
```

Comment:
```rust
use crate::task::{self, Poll};
use core::fmt;
use core::marker::{PhantomData, Unpin};
use core::mem::PinMut;
```

Note: `Context` should alway be referred to as `task::Context`

Reply:
Ah I see. We don't do so in libcore. Well, we should...

Reply:
The `core` instead of `std` will also fix the broken CI which tires to compiles futures-rs for cortex-m microchips and fails because it needs `no_std`